app/services/doc_auth/response.rb

```diff
@@ -7,6 +7,8 @@ class Response
       'Drivers License' => 'drivers_license',
     }
 
+    SUPPORTED_COUNTRY_CODES = %w[US GU VI AS MP PR USA GUM VIR ASM MNP PRI]
```

Comment:
You're adding this list two places, is there a way to keep it only in one place?

Reply:
@charleyf , was thinking that too and totally agree, but I was hesitate to introduce new classes, files based on my past experiences.

Reply:
@charleyf refactored it anyway, put it as a config item.